server/game/drawcard.js

```diff
@@ -458,7 +459,19 @@ class DrawCard extends BaseCard {
         this.saved = false;
     }
 
+    showFacedownTarget() {
+        this.revealWhenHidden = true;
```

Comment:
This approach will not work with Melee because it's all or nothing - it will show the card to all opponents, not just the one currently viewing cards. That works fine with False Spring, which explicitly reveals the cards, but not with His Viper's Eyes or Seen in Flames, as those are explicitly "look at" opponent's hand (only current player can see) not "reveal" opponent's hand (everyone can see)